src/bin/index.ts

```diff
@@ -110,13 +113,15 @@ async function run(args: CliArgs) {
     target,
     runtime,
     dts,
+    noDts,
     env,
     clean,
   } = args
   const cwd = args.cwd || process.cwd()
   const file = args.file ? path.resolve(cwd, args.file) : undefined
   const bundleConfig: BundleConfig = {
     dts,
+    noDts,
```

Comment:
merge with `dts`, by default dts is enabled for typescript, if cli dts is set to false, then we disable dts

Reply:
But currently, `--dts determine if need to generate types, default: false`

Reply:
it's only affecting CLI now, and for ts we're using other condition like hasTsConfig to determine if we should generate types.

when it's off, we should not generate any ts, so we also need to turn off somewhere.

I feel this option wasn't very well designed, it was used for only generating types. but ts project kinda ignore force re-enable it

Reply:
So, do we need to change the default option to true. Or, use undefined and explicit false to distinguish.

Reply:
I think we can leave the default to `undefined`, which means will still enable in ts. This is only a documentation change.

And we'll also check if dts is `false` when we decide to generate types